capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -1,17 +1,18 @@
 document.body.prepend(dynamicButton);
 
-function createClassroom() {
+function createClassroom(subject) {
     var auth2 = gapi.auth2.getAuthInstance();
     var name = "";
     var email = "";
+    var id_token = "";
     if (auth2.isSignedIn.get()) {
         name = auth2.currentUser.get().getBasicProfile().getName();
         email = auth2.currentUser.get().getBasicProfile().getEmail();
```

Comment:
Do we still need a user's email now that we have the web filter?

Reply:
Although the web filter was built, the next PR focuses on creating the client Header functionality so that the PR after that focuses on using the ID_TOKEN to create classrooms that are related to the user. For now, dummy users are still being used.